packages/utils/src/account.ts

```diff
@@ -75,12 +75,12 @@ export async function generateKeys(
 }
 
 export function generateDeterministicInviteCode(
```

Comment:
Once ARKE is available we plan to have users generate their own pepper (e.g. outside of ODIS). If we want this to be forward compatible, we should include in the hash the `phoneHash` to ensure that if two users pick the same pepper (potentially on purpose) the keys will still be unique.

Reply:
I'm not super familiar with this so could you please double check I'm hashing correctly?

Reply:
Looks good to me!